### PR TITLE
Add the Quick Start guide to the website

### PR DESCRIPTION
**Pull request by scholzj**
We should add the new Quick Start guide to our website.

### CODE REVIEW

File: docs/quickstart/master/full.html [hunk 1]
```diff
@@ -0,0 +1,953 @@
...
+<div id="footer">
+<div id="footer-text">
+Last updated 2019-10-04 19:12:44 CEST
+</div>
+</div>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+<script>hljs.initHighlighting()</script>
+</body>
+</html>
```
Comment (tombentley) [inline docs/quickstart/master/full.html]: This file is generated? Does it need to be added to git?

File: docs/quickstart/master/master.html [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+<p>Confirm that you see the incoming messages in the consumer console.</p>
+</li>
+<li>
+<p>Press Crtl+C to exit the Kafka console producer and consumer.</p>
+</li>
+</ol>
+</div>
+</div>
+</div>
+</div>
```
Comment (tombentley) [inline docs/quickstart/master/master.html]: Same question

File: docs/quickstart/master/full.html (hunk 1 above)
Comment (scholzj) [reply docs/quickstart/master/full.html]: The TravisCI build normally generates these files in master build and pushes them to the website repo automatically. However, for the initial commit we have to add them manually here, merge this and then we can merge the PR strimzi/strimzi-kafka-operator#2053 which will be pushing it from the master builds automatically. If we merge this PR without them the website will have broken links until the other PR is merged and the Travi build there completes.

File: docs/quickstart/master/master.html (hunk 2 above)
Comment (scholzj) [reply docs/quickstart/master/master.html]: See above.

**Pull request merged**